examples/with-apollo-auth/lib/withApollo.js

```diff
@@ -36,12 +43,6 @@ export default App => {
         appProps = await App.getInitialProps(ctx)
```

Comment:
Isn't the reason this check is there that `App.getInitialProps` (and pages) could finish the request like this. Though it doesn't really matter since if the request is sent/finished Next.js will detect it and not start rendering.

Reply:
so we can probably completly get rid of this check (res.finished part), if so I can then remove it completly?

Reply:
Well it's correct to have it where it was already located. Since if a page redirects, you don't want to fetch apollo data for the page (additional overhead)

Reply:
Ok then I close this one :)